### PR TITLE
mbf21: weapons

### PR DESCRIPTION
**Pull request by rfomin**
See: https://github.com/rfomin/woof/blob/9e3e303d5eed7d9e0b2c72387966313bd37779bb/docs/mbf21.md#weapons

### CODE REVIEW

File: Source/d_deh.c [hunk 1]
```diff
@@ -46,6 +46,7 @@
 #include "../win32/win_fopen.h"
 #endif
 
+//static boolean bfgcells_modified = false;
```
Comment (fabiangreffrath) [inline Source/d_deh.c:49]: What's the matter with this ominous commented variable?

File: Source/p_inter.c [hunk 2]
```diff
@@ -80,6 +80,34 @@ int clipammo[NUMAMMO] = { 10,  4,  20,  1};
 // Returns false if the ammo can't be picked up at all
 //
 
+static boolean P_GiveAmmoAutoSwitch(player_t *player, ammotype_t ammo, int oldammo)
```
Comment (fabiangreffrath) [inline Source/p_inter.c:83]: New code block needs a comment, please.

**Review by fabiangreffrath — CHANGES_REQUESTED** [review]

File: Source/d_deh.c (hunk 1 above)
Comment (rfomin) [reply Source/d_deh.c]: I forgot to remove it.

**Comment by fabiangreffrath** [comment]
Thank you!

**Pull request merged**